packages/vscode-extension/package.json

```diff
@@ -1303,6 +1314,7 @@
     "url-loader": "^4.1.1",
     "uuid": "^8.3.2",
     "verdaccio": "^5.18.0",
+    "vscode-extension-tester": "^5.5.1",
```

Comment:
Why merging UI test into *packages/vscode-extension* instead of creating a new one such as *packages/vsc-ui-test*?

IMO, UI test only depends on the output visx file so no need to merge source code together.
Merging UI test into *vscode-extension* may bring extra dev effort, e.g. dependency compatibility between the real source code and the ui test code.

In addition, `vscode-extension-tester`'s LICENSE is different.

Reply:
create the new `packages/vsc-ui-test` make sense.
`vscode-extension-tester`'s LICENSE is Apache 2.0, later I will run Compliance pipeline on ADO to scan this dependency.

Reply:
@swatDong 
https://dev.azure.com/mseng/VSIoT/_build/results?buildId=20272402&view=results
I runed the Static Code Analysis for TeamsFx on ADO, and the result is success.

Reply:
BTW, is there any concern since the license of our code repo is MIT. I have little guidance here but better to check with PM.